exalearn-ui/src/app/pages/test-page/test-page.component.ts

```diff
@@ -1,18 +1,35 @@
 import { Component, OnInit } from '@angular/core';
 import { questions } from '../../test-data/test-questions';
-import { Question } from '../../interfaces/interfaces';
+// import { Question } from '../../interfaces/interfaces';
+import { Question } from '../../interfaces/question.interface';
+import { TestsService } from '../../services/tests.service';
 
 @Component({
 	selector: 'app-test-page',
 	templateUrl: './test-page.component.html',
-	styleUrls: ['./test-page.component.scss']
+	styleUrls: ['./test-page.component.scss'],
+	providers: [TestsService]
 })
 export class TestPageComponent implements OnInit {
-	testQuestions: Question[] = [];
+	// testQuestions: Question[] = [];
+	level: string = 'beginner';
```

Comment:
I think we have enums.ts with english level on dev.
Can you use them unstead?

Reply:
I just wanted to be consistent with the mockup. Yes we can use enum, can be refactored.